sdk/firmware.ldscript.in

```diff
@@ -31,6 +31,10 @@ SECTIONS
 		*/cherimcu.allocator.compartment(.compartment_export_table);
 		.allocator_export_table_end = .;
 
+		.token_server_export_table = ALIGN(8);
+		*/cherimcu.token_server.compartment(.compartment_export_table);
```

Comment:
At some point soon, I intend to do an s/cherimcu/cheriot/ in the xmake for these things.  Can you do this preemptively for the new things so that we don't regress?